lib/xdp_loader.h

```diff
@@ -0,0 +1,31 @@
+#ifndef __XDP_LOADER_H_
+#define __XDP_LOADER_H_
+
+using namespace std;
```

Comment:
You should not alias namespaces in header files unless you are exporting a specific function or variable to consumers of your header.

Reply:
Done